content/blog/benefits-of-policy-as-code/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Benefits of Policy as Code"
+date: 2020-05-27
+meta_desc: "An organization can benefit from Policy as Code through cost control, compliance, and best practices."
+meta_image: pac-benefits.png
+authors:
+    - sophia-parafina
+tags:
+    - policy-as-code
+---
+
+Writing policies in a high-level programming language can automate the process. When policies are written with code, you can apply software development practices such as testing, automated deployment, and version control. Cloud providers typically offer a GUI to create policies,  but creating policies is not easily repeatable, nor can you version policies. Moreover, policies must be tested against a live system, which means using an existing system or configuring and deploying an ephemeral version.
+
+While the benefits of writing policies as code are evident for developers and operators, the organizational benefits are even more significant. Organizations can realize cost savings, improved compliance, efficient deployments, fine-grained control over infrastructure, and better use of cloud provider native resources. Let’s take a look at these benefits in-depth.
+
+<!--more-->
+
+## Controlling Costs
+
+A sizeable monthly cloud bill due to unused resources left running or using over-sized instances for small tasks is an industry cliché. One way to control costs is to set policies based on pricing. With pricing data online, we can calculate the cost of a resource ahead of time and create a policy that limits the amount spent to deploy it. In a [previous article]({{< relref "/blog/manage-infrastructure-with-pac#controlling-cost-on-aws" >}})), we provided an example policy that finds all the resources in a deployment and calculates the total monthly cost. The policy sets the monthly cost permitted and sends a warning if the deployment exceeds it.
+
+But what about unused or abandoned resources? We can use a cloud provider’s resources to implement a watchdog function with a policy that cleans up unused resources. In this [article]({{< relref "/blog/controlling-aws-costs-with-lambda-and-pulumi" >}}), we show how to use a serverless function to find unused resources daily and shut them down. We implemented this system at Pulumi and [reduced our operating costs by 64%]({{< relref "/blog/controlling-aws-costs-with-lambda-and-pulumi#in-conclusion" >}}).
+
+## Compliance
+
+Policy as Code is a way to enforce infrastructure policies that prevent inadvertent access to resources such as databases and storage or to enforce cost policies.
```

Comment:
For the heading Compliance, it might be better to move the cost stuff to the previous section and instead talk about networking policies. [Just a suggestion]

Reply:
Updated this section and moved the cost part to previous section